test/openAPI/test-data.json

```diff
@@ -207,6 +242,22 @@
       "Gov_Stack_BB": "example Gov_Stack_BB",
       "voucher_status": "Blocked",
       "voucher_amount": "1000"
+    },
+    "// already_used",
+    {
+      "voucher_number": "60011",
+      "voucher_serial_number": 6001,
+      "Gov_Stack_BB": "Gov_Stack_BB",
+      "voucher_status": "Blocked",
```

Comment:
@AKibuuka I've added data here for vouchers: one that should already be used and for one that should be expired, but I'm not sure what should be the `voucher_status` value for them. For now i set it to `Blocked` but if it should be changed please let me know. 

cc. @KarolinaKopacz